guides/source/association_basics.md

```diff
@@ -788,7 +788,23 @@ module MyApplication
 end
 ```
 
-This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope. But the following will _not_ work, because `Supplier` and `Account` are defined in different scopes:
+This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope. This is highly beneficial as it allows you to organize your models according to their scope. For example:
```

Comment:
Describing the actual scope (`MyApplication::Business`) would help here I think.
We can also move the comment about "folders" here to help the reader notice what to look for.
```suggestion
This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope (`MyApplication::Business`). This organization allows structuring models into folders based on their scope:
```